_data/harnesses/kafka-producer-application/kafka.yml

```diff
@@ -0,0 +1,166 @@
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/create-topic.adoc
+
+    - title: Configure the project
+      content:
+        - action: make_file
+          file: build.gradle
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/make-build-file.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/gradle-wrapper.sh
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/make-gradle-wrapper.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/make-configuration-dir.sh
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/make-config-dir.adoc
+
+        - action: make_file
+          file: configuration/dev.properties
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/make-dev-file.adoc
+
+    - title: Create the KafkaProducer application
+      content:
+        - action: execute
+          file: tutorial-steps/dev/make-src-dir.sh
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/make-src-dir.adoc
+            
+        - action: make_file
+          file: src/main/java/io/confluent/developer/KafkaProducerApplication.java
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/make-application.adoc 
+
+    - title: Create data to produce to Kafka
+      content:
+        - action: make_file
+          file: input.txt
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/create-data-file.adoc        
+
+    - title: Compile and run the KafkaProducer application
+      content:
+        - action: execute
+          file: tutorial-steps/dev/build-uberjar.sh
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/build-uberjar.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/run-dev-app.sh
+          stdin: tutorial-steps/dev/input-harness-filename.txt
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/run-dev-app.adoc
+
+    - title: Confirm records sent by consuming from topic
+      content:
+        - action: execute_async
+          file: tutorial-steps/dev/harness-console-consumer.sh
+          stdout: tutorial-steps/dev/outputs/actual-output.txt
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/run-consumer.adoc
+
+        - name: wait for the consumer to read the messages
+          action: sleep
+          ms: 5000
+          render:
+            skip: true
+
+test:
+  steps:
+    - title: Create a test configuration file
+      content:
+        - action: make_file
+          file: configuration/test.properties
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/test/make-test-file.adoc
+
+    - title: Write a test
```

Comment:
FYC would this be appropriate to name as `Write a unit test` (note term `unit` is added)

Reply:
ack